quic/s2n-quic-platform/src/socket/task/events.rs

```diff
@@ -73,19 +69,11 @@ impl crate::syscall::SocketEvents for TxEvents {
                 // if we got interrupted break and have the task try again
                 ControlFlow::Break(())
             }
-            #[cfg(s2n_quic_platform_gso)]
-            _ if error.raw_os_error() == Some(libc::EIO) => {
+            _ => {
                 // on platforms that don't support GSO we need to disable it and mark the packet as
                 // "sent" even though we weren't able to.
```

Comment:
is this comment still accurate?

Reply:
i think it still is... it's just saying why we're calling the `on_socket_error`. but i can change it to something else if you'd prefer.